x/ecocredit/core/msg_update_project_admin_test.go

```diff
@@ -0,0 +1,79 @@
+package core
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"gotest.tools/v3/assert"
+)
+
+func TestMsgUpdateProjectAdmin_ValidateBasic(t *testing.T) {
+	addr := sdk.AccAddress("addr1-----------------").String()
+	addr2 := sdk.AccAddress("addr2-----------------").String()
```

Comment:
A valid address will still be generated without `-----------------`.

```suggestion
	addr := sdk.AccAddress("addr1").String()
	addr2 := sdk.AccAddress("addr2").String()
```